packages/connect-ui/package.json

```diff
@@ -9,7 +9,11 @@
         "preview": "vite preview"
     },
     "devDependencies": {
+        "@nangohq/types": "0.42.10",
```

Comment:
Should add this to the publish.sh `bump_other_pkg` https://github.com/NangoHQ/nango/blob/master/scripts/publish.sh#L83

Reply:
oh yeah thanks 🙏🏻